tests/unit/query_graph/conftest.py

```diff
@@ -130,6 +130,8 @@ def input_node_fixture(global_graph, input_details):
         "columns": [
             {"name": "ts", "dtype": DBVarType.TIMESTAMP},
             {"name": "cust_id", "dtype": DBVarType.INT},
+            {"name": "biz_id", "dtype": DBVarType.INT},
+            {"name": "product_type", "dtype": DBVarType.INT},
```

Comment:
Adding these columns in the input node changes the aggregation_id in the tests. Previously, tile compute queries in the tests referenced non-existent columns as groupby keys. With this PR, the columns now need to exist due to changes in how the queries are constructed.